addons/account/tests/test_account_move_out_invoice.py

```diff
@@ -3294,6 +3294,8 @@ def _check_invoice_values(invoice):
         _check_invoice_values(invoice_create)
 
         move_form = Form(self.env['account.move'].with_context(default_move_type='out_invoice'))
+        # Allow the journal to be editable
+        move_form.name = False
```

Comment:
Why is this needed for a change in the view?

Reply:
They emulate the form view so the read only on the view matter

Reply:
Oh, I see.
But it should still be editable when `posted_before = False` because you cannot really create a gap in that case; only make the sequence start at 2 instead of 1 (which is not an issue in most countries)

Reply:
Do we need to change the domain then ? not sure if you are asking for a change ahah

Reply:
yes